standalone-metastore/metastore-common/src/gen/thrift/gen-javabean/org/apache/hadoop/hive/metastore/api/hive_metastoreConstants.java

```diff
@@ -73,7 +73,7 @@
 
   public static final java.lang.String JDBC_CONFIG_PREFIX = "hive.sql.";
 
-  public static final java.lang.String TABLE_IS_CTAS = "created_with_ctas";
+  public static final java.lang.String TABLE_IS_CTAS_OR_CMV = "created_with_ctas";
```

Comment:
what would happen if we regenerate from thrift? we should either keep it as it was or create a new const for MV

Reply:
Reverted this change. I prefer keeping it as it was.

Reply:
👍